content/starters/index.md

```diff
@@ -12,17 +12,22 @@
 
 
 - If you **Deploy with Netlify**, a new repository will be created on your GitHub account and it will be deployed to [Netlify](https://www.netlify.com/). Later you can make changes in the deployed website by  [Committing to the New Repository](https://docs.github.com/en/github/managing-files-in-a-repository/adding-a-file-to-a-repository-using-the-command-line). (This method is recommended to quickly get a blog running and deployed).
-- If you **Create Project Locally** with `create-abell-app`, You will get a website running locally. Later you can deploy to whichever platform you like.
+- If you **Create Project Locally** with `create-abell-app`, You'll get a website running locally. Later, you can deploy to the platform of your choice.
 
 ---
-## Table of Content
-
-- [Setting Up Project Locally](#setting-up-project-locally)
-- [Folder Structure](#folder-structure)
-- [Edit/Create/Update Post](#editcreateupdate-post)
+## Table of Contents
+
+- [{{ meta.title }}](#-metatitle-)
+  - [Starters](#starters)
+  - [Table of Contents](#table-of-contents)
```

Comment:
The starters are at the top before the table of contents so the starters are not exactly the part of the document that comes after this. This can be a little confusing I guess? Someone may read "Starters" here and search for that heading below but it won't exist. Maybe we can remove Starters and Table of Contents from here?